domains/ramateja.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+      "username": "ramateja99"
```

```suggestion
      "username": "ramateja99",
```
the comma should be here